certified-connectors/IronDefense/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2246 @@
+{
+  "securityDefinitions": {
+    "basic": {
+      "type": "basic"
+    }
+  },
+  "security": [
+    {
+      "basic": [],
+      "jwt": []
+    }
+  ],
+  "swagger": "2.0",
+  "info": {
+    "title": "IronNet IronAPI",
+    "description": "IronNet’s IronAPI allows clients to interface with the IronDefense backend through a curated set of endpoints. This provides customers with the capability to triage alerts outside of IronVue.",
+    "version": "3.4.9",
+    "contact": {
+      "name": "IronNet Support",
+      "url": "https://ironnet.com",
+      "email": "support@ironnet.com"
+    },
+    "license": {
+      "name": "Commercial License",
+      "url": "https://ironnet.com"
```

Comment:
@marcus-defazio It would be better to provide a web page which says your company has this commercial license.

Reply:
Changed to a web page with license info